app/Models/Foundation/Summit/Events/Presentations/Presentation.php

```diff
@@ -2402,4 +2418,77 @@ public function getSnapshot():array{
 
         return array_merge( $snapshot, parent::getSnapshot());
     }
+
+    /**
+     * @return string
+     * @throws \Exception
+     */
+    public function getReviewStatus(): string
+    {
+        if ($this->selection_plan == null) return self::ReviewStatusNoSubmitted;
```

Comment:
@romanetar please lets not change the logic , lets be sure to follow the same logic as https://github.com/fntechgit/call-for-presentations/blob/b1952301be6962feb89913448bfdd4543c527823/src/model/presentation.js#L84